model/Vocabulary.php

```diff
@@ -229,6 +229,21 @@ public function getDefaultConceptScheme()
         return array_pop($conceptSchemeURIs);
     }
 
+    /**
+     * Return the URI of the default concept scheme of this vocabulary, as set in the vocabulary configuration, or null
+     * if not set.
+     * @return string default concept scheme URI, or null if undefined
+     **/
+
+    public function getMainConceptScheme(){
```

Comment:
This is partially duplicating `getDefaultConceptScheme`. Could you also make `getDefaultConceptScheme` take advantage of this function to avoid the duplication. (i.e. if getMainConceptScheme() returns null, guess the concept scheme)